docusaurus.config.js

```diff
@@ -60,17 +60,17 @@ const config = {
       navbar: {
         title: 'RangL',
         logo: {
-          alt: 'My Site Logo',
-          src: 'img/logo.svg',
+          alt: 'RangL Logo',
+          src: 'img/rangl-logo.png',
         },
         items: [
-          {
-            type: 'doc',
-            docId: 'intro',
-            position: 'left',
-            label: 'Docs',
-          },
```

Comment:
Yes! @moriartyjm mentioned that it would be better to link the challenges to the EvalAI site. Only exception is with VGI, that's why I left it out in a single page outside the docs

Reply:
No worries. I'll try make it so that we can get this to happen without the Netlify build complaining.